src/geometry/WorkingGeometry.ts

```diff
@@ -83,7 +100,7 @@ export class WorkingGeometry {
             return vec3.fromValues(face.normal[0], face.normal[1], face.normal[2]);
         });
 
-        // Make all of the baked shapes red for now.
+        // Make all of the baked shapes red for now
```

Comment:
nit: Can you add back the period?

Reply:
I noticed that all the other single line comments in the code base had no period after the comment. I opted for consistency. If we want to change that I'd rather change them all in a separate PR.

Reply:
I'd say not to worry about it. As long as the doc comments are ok (since that may plausibly be user-facing) I'm cool with anything